### ISSUE TITLE
invalid workflow definition

### ISSUE THREAD

**Issue by rluvaton** [fastify/workflows#82]
the last change to `plugin-ci.yml` is invalid:
- #80

```
he workflow is not valid. In .github/workflows/ci.yml (Line: 20, Col: 11): Error from called workflow fastify/workflows/.github/workflows/plugins-ci.yml@v3 (Line: 104, Col: 11): A mapping was not expected
```

> source:
> - https://github.com/fastify/fastify-reply-from/pull/319#issuecomment-1583206153

I'll create a PR here to add a validator for the GitHub action but @mcollina can you please take a look

**Comment by mcollina** [comment]
thanks!

**Comment by rluvaton** [comment]
NP, I fixed the workflow file in #83 and added the validation in #84
I suggest first merging the workflow file, enabling it, and testing if it's passing in #84 and failing on master

**Comment by rluvaton** [comment]
Fixed
